packages/flutter/test/material/search_test.dart

```diff
@@ -1034,20 +1055,31 @@ void main() {
     expect(rootObserver.pushCount, 0);
     expect(localObserver.pushCount, 0);
 
-    // showSearch normal and back
+    // showSearch normal and back.
```

Comment:
Can we add a check here if the back button is showing as expected.

Reply:
there was no code I could add to this line. It came from the current master version from another developer. I have added minus when I see it is not in use. Now the checking back button is visible. thank you